manage

```diff
@@ -487,7 +487,7 @@ if ! isInstalled ${DOCKER_EXE}; then
   exit 1
 fi
 
-DOCKER_COMPOSE_EXE="${DOCKER_EXE} compose"
+DOCKER_COMPOSE_EXE="${DOCKER_EXE}-compose"
```

Comment:
This is specifically using the new `docker compose` syntax, rather than the legacy `docker-compose` syntax.  Please revert this change.

Reply:
But then it is explicitly using the old syntax that doesn’t work everywhere either.  Better still to use the change [here](https://github.com/hyperledger/aries-agent-test-harness/blob/3b8b2eb8f74088bcc6bcbe5457880dbe1ff1ea25/manage#L69) to use the command appropriate for the version of docker in use.

Reply:
To clarify the PR changed the call from the new `docker compose` syntax to the legacy `docker-compose` syntax.  This project has never used the legacy `docker-compose` syntax.  If I recall, I could be wrong, but I believe there are some features in the scripts that require the new version of docker compose.  There was a reason I started with it, I just can't recall exactly.